test/data/test_multi_nested_tensor.py

```diff
@@ -81,6 +81,30 @@ def test_multi_nested_tensor_basics(device):
     assert dense_multi_nested_tensor.shape == (multi_nested_tensor.shape[:-1] +
                                                (max_len, ))
 
+    multi_nested_tensor_with_nan = multi_nested_tensor.clone()
+    multi_nested_tensor_with_nan.values[0::2] = -1
+
+    # Test fill_col
+    multi_nested_tensor_1 = multi_nested_tensor_with_nan.clone()
+    for col in range(multi_nested_tensor_with_nan.num_cols):
+        multi_nested_tensor_1.fillna_col(col, 100)
+    assert not torch.all(multi_nested_tensor_1.values == -1).any()
+
+    # Test fillna
+    multi_nested_tensor_2 = multi_nested_tensor_with_nan.clone()
+    multi_nested_tensor_2.fillna_(100)
+    assert not torch.all(multi_nested_tensor_2.values == -1).any()
+
+    # Test fillna with vector of shape (num_cols)
+    multi_nested_tensor_3 = multi_nested_tensor_with_nan.clone()
+    multi_nested_tensor_3.fillna_(torch.arange(100, 110))
+    assert torch.all(
+        torch.eq(multi_nested_tensor_3.offset, multi_nested_tensor.offset))
```

Comment:
I don't think we need to include this extra functionality, unless we use it in our main codebase.

Reply:
Sure.